ssl/t1_lib.c

```diff
@@ -3170,10 +3170,11 @@ static int tls_decrypt_ticket(SSL *s, const unsigned char *etick,
     p = sdec;
 
     sess = d2i_SSL_SESSION(NULL, &p, slen);
+    slen -= p - sdec;
```

Comment:
In some cases, that will still raise warnings.  We have a `PTRDIFF_T` in `crypto/evp/evp_enc.c` that we probably should make wider use of for these kinds of things.